eve/main.yml

```diff
@@ -76,6 +76,24 @@ stages:
         doStepIf: '%(prop:launch_end2end)s'
         stage_names:
         - end2end
+    - ShellCommand:
```

Comment:
feels like a bit of work as a pre-requisite to promoting... 😏

Reply:
yes... it does :man_facepalming: